update NFT page (#7083)
ethereum/ethereum-org-website: src/content/nft/index.md

  - Signatures
- Lots and lots more options to get creative with!

> Ownership of NFTs is managed through the uniqueID and metadata that no other token can replicate. NFTs are minted through smart contracts that assign ownership and manage the transferability of the NFT's. When someone creates or mints an NFT, they execute code stored in smart contracts that conform to different standards, such as ERC-721. This information is added to the blockchain where the NFT is being managed. The minting process, from a high level, has the following steps that it goes through:

Reviewer: Side note, I think for a non-technical page we should just use "unique ID" instead of concatenating like a variable name: 
```suggestion
Ownership of NFTs is managed through the unique ID and metadata that no other token can replicate. NFTs are minted through smart contracts that assign ownership and manage the transferability of the NFT's. When someone creates or mints an NFT, they execute code stored in smart contracts that conform to different standards, such as ERC-721. This information is added to the blockchain where the NFT is being managed. The minting process, from a high level, has the following steps that it goes through:
```

Also, perhaps we link to the ERC-721 docs page? Open question since this isn't a technical page, feel free to ignore this:

```suggestion
Ownership of NFTs is managed through the unique ID and metadata that no other token can replicate. NFTs are minted through smart contracts that assign ownership and manage the transferability of the NFT's. When someone creates or mints an NFT, they execute code stored in smart contracts that conform to different standards, such as [ERC-721](/developers/docs/standards/tokens/erc-721/). This information is added to the blockchain where the NFT is being managed. The minting process, from a high level, has the following steps that it goes through:
```

#### Minting NFTs {#minting-nfts}

When you mint an NFT, a few things have to happen:

- It needs to be confirmed as an asset on the blockchain.
- The owner's account balance must be updated to include that asset. This makes it possible for it to then be traded or verifiably "owned".
- The transactions that confirm the above need to be added to a block and "immortalized" on the chain.
- The block needs to be confirmed by everyone in the network as "correct". This consensus removes the need for intermediaries because the network agrees that your NFT exists and belongs to you. And it's on chain so anyone can check it. This is one of the ways Ethereum helps NFT creators to maximize their earnings.

All these tasks are done by block producers and validators. Block proposers add your NFT transaction to a block and broadcast it to the rest of the network. Validators check that the transaction is valid and then add it to their databases. There are lots of crypto-economic incentives in place to make sure validators are acting honestly. Otherwise, anyone could just claim that they own the NFT you just minted and fraudulently transfer ownership.

#### NFT security {#nft-security}

> Ethereum's security comes from its consensus mechanism which uses proof-of-stake. Validators deposit 32 ETH into a contract that can be destroyed if they misbehave. The system is designed to strongly economically disincentivize malicious actions, making Ethereum effectively tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ETH to change it. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT and the bad actor would be economically penalized and ejected from the pool of validators to prevent it from happening.

Reviewer 2: I don't think we need to talk about validators depositing ETH here to make this point

```suggestion
Ethereum's security comes from proof-of-stake. The system is designed to economically disincentivize malicious actions, making Ethereum tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ETH to change it. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT, and the bad actor would be economically penalized and ejected.
```

Secure meaning no one can copy/paste your NFT or steal it.

> These qualities of Ethereum makes digitally owning unique items and getting a fair price for your content possible. Ethereum protects the assets using a decentralized consensus mechanism called ['proof-of-stake'](/developers/docs/consensus-mechanisms/pos). This is a low carbon consensus mechanism that is also considered more secure than the energy-intensive alternative, ['proof-of-work'](/developers/docs/consensus-mechanisms/pow). NFTs have been associated with high energy expenditure because Ethereum used to be secured using proof-of-work. This is no longer true.

Reviewer: If possible, I think we should try to avoid strictly calling proof-of-stake a consensus mechanism as it is just one piece of consensus to determine who is a valid block producer.
```suggestion
These qualities of Ethereum makes digitally owning unique items and getting a fair price for your content possible. Ethereum protects the assets using a decentralized consensus mechanism which involves ['proof-of-stake'](/developers/docs/consensus-mechanisms/pos). This is a low carbon method to determine who can add a block of transactions to the chain, and is considered more secure than the energy-intensive alternative, ['proof-of-work'](/developers/docs/consensus-mechanisms/pow). NFTs have been associated with high energy expenditure because Ethereum used to be secured using proof-of-work. This is no longer true.
```

> - The transactions that confirm the above need to be added to a block and "immortalised" on the chain.

Reviewer: ```suggestion
- The transactions that confirm the above need to be added to a block and "immortalized" on the chain.
```

> - The block needs to be confirmed by everyone in the network as "correct". This consensus removes the need for intermediaries because the network agrees that your NFT exists and belongs to you. And it's on chain so anyone can check it. This is one of the ways Ethereum helps NFT creators to maximise their earnings.

Reviewer: ```suggestion
- The block needs to be confirmed by everyone in the network as "correct". This consensus removes the need for intermediaries because the network agrees that your NFT exists and belongs to you. And it's on chain so anyone can check it. This is one of the ways Ethereum helps NFT creators to maximize their earnings.
```

All these tasks are done by block producers and validators. Block proposers add your NFT transaction to a block and broadcast it to the rest of the network. Validators check that the transaction is valid and then add it to their databases. There are lots of crypto-economic incentives in place to make sure validators are acting honestly. Otherwise, anyone could just claim that they own the NFT you just minted and fraudulently transfer ownership.

#### NFT security {#nft-security}

> Ethereum's security comes from its proof-of-stake consensus mechanism - validators deposit 32 ETH into a contract that can be destroyed if they misbehave. The system is designed to strongly economically disincentivize malicious actions, making Ethereum effectively tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ether to change it, and even then their success is not guaranteed because the Ethereum community could decide to fork their malicious actions away. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT and the bad actor would be economically penalized and ejected from the pool of validators to prevent it from happening.

Reviewer: ```suggestion
Ethereum's security comes from its consensus mechanism which uses proof-of-stake. Validators deposit 32 ETH into a contract that can be destroyed if they misbehave. The system is designed to strongly economically disincentivize malicious actions, making Ethereum effectively tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ETH to change it, and even then their success is not guaranteed because the Ethereum community could decide to fork their malicious actions away. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT and the bad actor would be economically penalized and ejected from the pool of validators to prevent it from happening.
```

Also:

> even then their success is not guaranteed because the Ethereum community could decide to fork their malicious actions away. 

Just noting this as it could potentially cause confusion... do we want to mention this here?

Author: yea i suppose it is a bit too nuanced a discussion for this page. 
```suggestion
Ethereum's security comes from its proof-of-stake consensus mechanism - validators deposit 32 ETH into a contract that can be destroyed if they misbehave. The system is designed to strongly economically disincentivize malicious actions, making Ethereum effectively tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ether to change it. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT and the bad actor would be economically penalized and ejected from the pool of validators to prevent it from happening.
```

- [A beginner's guide to NFTs](https://linda.mirror.xyz/df649d61efb92c910464a4e74ae213c4cab150b9cbcc4b7fb6090fc77881a95d) – _Linda Xie, January 2020_
- [Everything you need to know about the metaverse](https://foundation.app/blog/enter-the-metaverse) – _Foundation team, foundation.app_
- [No, CryptoArtists Aren’t Harming the Planet](https://medium.com/superrare/no-cryptoartists-arent-harming-the-planet-43182f72fc61)

> - [Ethereum's energy consumption](/energy_consumption)

Reviewer 2: ```suggestion
- [Ethereum's energy consumption](/energy-consumption/)
```
